pyproject.toml

```
@@ -0,0 +1,51 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[tool.poetry]
+name = "pixels"
+version = "1.0.0"
+description = "A collaborative canvas, where users paint pixel by pixel, by POSTing co-ordinates and a colour code to an API."
```

```suggestion
description = "A collaborative canvas, where users paint pixel by pixel, by POSTing coordinates and a colour code to an API."
```
The hyphen has been widely dropped in most places